config/initializers/okcomputer.rb

```diff
@@ -121,7 +121,9 @@ def check
 OkComputer::Registry.register 'external-workflow-services-url', OkComputer::HttpCheck.new(workflows_url)
 
 # Replication (only) uses zip_storage directory to build the zips to send to zip endpoints
-OkComputer::Registry.register 'feature-zip_storage_dir', OkComputer::DirectoryCheck.new(Settings.zip_storage)
+unless Settings.resque_dashboard_hostnames.include?(Socket.gethostname) # the dashboard hosts do not mount the directory
```

Comment:
this comment is not clarifying things for me.  I wonder if we'd be better off just hard-coding that it's -01 that is the problem child here and explaining why.

Reply:
@ndushay I'll replace the one-liner comment with a sentence or two. I'd rather use what's in settings than stuff explicit hostnames in the initializer, personally. This was also what @jmartin-sul was inclined to do.

Reply:
@ndushay Wrote up a little paragraph. Take another look?